Latin Abbreviations "e.g." Updated To "for example" (#19182)
kubernetes/website: content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md

{{< feature-state state="stable" for_kubernetes_version="1.16" >}}

> CustomResourceDefinitions traditionally store any (possibly validated) JSON as is in etcd. This means that unspecified fields (if there is a [OpenAPI v3.0 validation schema](/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#validation) at all) are persisted. This is in contrast to native Kubernetes resources like for example a pod where unknown fields are dropped before being persisted to etcd. We call this "pruning" of unknown fields.

Reviewer: ```suggestion
CustomResourceDefinitions traditionally store any (possibly validated) JSON as is in etcd. This means that unspecified fields (if there is a [OpenAPI v3.0 validation schema](/docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions/#validation) at all) are persisted. This is in contrast to native Kubernetes resources such as a pod where unknown fields are dropped before being persisted to etcd. We call this "pruning" of unknown fields.
```
